src/lib/pka_queue.c

```diff
@@ -155,6 +155,33 @@ static uint16_t pka_concat6_wlen(uint32_t word_len,
     return total_wlen;
 }
 
+// Adjusts the value present in the array pointed by buf_ptr.
+// Adjustment is done by removing zero padding at the most significant bytes.
+// Once this is done, align the new length around 8-byte boundary.
+// Note: Here it's assumed that the value is in Little Endian format.
+static void pka_adjust_rslt(pka_operand_t *result)

```

Comment:
I believe "adjust" is a bit confusing. Could you please call it something like: pka_queue_no_zero_pad_rslt or any other name that explicitly indicates the type of the operation?

Note that, by convention, all functions in pka_queue.c have the prefix pka_queue_*

Reply:
Yes, I'll replace "adjust" with something more relevant. However, as discussed, the inclusion of "queue" in the function name is not necessary as this is a static function. Thanks